nc/tests/test_api.py

```diff
@@ -193,7 +209,8 @@ def test_contraband_hit_rate(self):
 
         searches = response.data['searches']
         self.assertEqual(searches[0]['year'], 2010)
-        self.assertEqual(searches[0][race_label], 2)
+        self.assertEqual(searches[0][race_label], 1)
+        self.assertEqual(searches[0]['hispanic'], 1)
         self.assertEqual(searches[1]['year'], 2012)
         self.assertEqual(searches[1][race_label], 1)
```

Comment:
Can we also add a few lines below these testing `response.data['contraband']`?

Reply:
That is already there, lines 218 - 221.

Reply:
Good point